github_api/prs.py

```diff
@@ -71,7 +72,8 @@ def formatted_votes_summary(votes, total, threshold):
     vfor = sum(v for v in votes.values() if v > 0)
     vagainst = abs(sum(v for v in votes.values() if v < 0))
 
-    return "with a vote of {vfor} for and {vagainst} against, with a weighted total of {total:.1f} and a threshold of {threshold:.1f}" \
+    return "with a vote of {vfor} for and {vagainst} against, with a weighted total \
+            of {total:.1f} and a threshold of {threshold:.1f}" \
```

Comment:
I think I can live with the longer max line length, but I'd prefer to still use parentheses as [PEP 8 recommends](https://www.python.org/dev/peps/pep-0008/#maximum-line-length).